input/docs/handbook/dependency-inversion/custom-dependency-inversion.md

```diff
@@ -0,0 +1,67 @@
+# Override Default Depenedency Inversion Container
+
+We understand that some developers would prefer to use their favorite dependency inversion container.  ReactiveUI allows for this by implementing `IMutableDependencyResolver`.  Once this class has been implemented you simply need to set it to the `Locator.
+
+## Implement an `IMuatableDependencyResolver`
+
+```csharp
+public abstract class MyDependencyResolver : IMutableDependencyResolver
+{
+    public abstract object GetService(Type serviceType, string contract = null);
+
```

Comment:
hmm.. may want to provide a reference implementation to having a Dictionary even of items etc.

Reply:
Or maybe we should provide a production-ready custom dependency resolver implementation for a trendy DI container, like <a href="https://github.com/simpleinjector/SimpleInjector">SimpleInjector</a> or < href="https://bitbucket.org/dadhi/dryioc">DryIoc<a> that are loved by mobile developers for their performance 🤔

Reply:
Specifically for the `GetService` method or for the entire implementation.  I struggled with this primarily because I don't want to affect peoples design decisions too much.  I could technically just rip the guts from [ModernDependencyResolver](https://github.com/reactiveui/splat/blob/master/src/Splat/ServiceLocation.cs#L227-L329) that way if they copy/paste from the website, they will at least be baseline.

Reply:
@worldbeater I actually wrote this last night while I was implementing this for [Autofac](https://github.com/autofac/Autofac).  I could provide that.

Reply:
@rlittlesii it'be great! Autofac is an extremely popular DI container so we may attract more people by saying RxUI supports it

Reply:
Changed to an implementation of an `AutofacDependencyResolver`